docs/source/hardware/low_priority_faqs.md

```diff
@@ -0,0 +1,210 @@
+::::
+
+::::{dropdown} 3. Are there policies or other formal documents available?
+RC is currently drafting a memorandum of understanding (MoU) that will be distributed to all private partitions owners later in the Fall semester 2023. This document will formalize the understanding that was stated to all private partitions owners at the time of purchase, since 2019. Moreover, RC is working on a detailed set of documentation, service level objectives and training sessions that will be available later in the Fall 2023 semester, once the service is ready to be released.
+::::
+
+::::{dropdown} 4. I have resources I would like to make available to the rest of the research community, what can I do?
+
+All hardware purchased on or after 2019 will be part of the `lowpriority` partition. Feel free to contact us if you purchased hardware before 2019 and would like to make it available to the `lowpriority` partition.
+::::
+
+::::{dropdown} 5. Why should I make my private resources available to the rest of the research community?
+
+By making your private resources available to others through the `lowpriority` partition, you will allow the Northeastern research community to use them when they are idle. This, in time, could double the resources available to NURC users and is a common practice in HPC clusters to optimize the use of idle private resources that consume power and cooling. 
+::::
+
+:::{button-link} faq.html
+:color: primary
+:align: center
+Back to the top
+:::
+
+## Testing and Release
+
+::::{dropdown} 1. Will there be a testing and review phase?
+We are currently looking for PIs who own private hardware and would like to be part of our testing phase. Once the testing phase is over RC will present the finalized service to the RCAC and all private partition owners for review.
+::::
+
+::::{dropdown} 2. What is the timeline of the testing phase?
+The `lowpriority` partition will be available for testing in Fall 2023. Only groups who opt in will have access to the `lowpriority` partition during the testing phase and only hardware owned by those groups will be part of the test partition.
+::::
+
+::::{dropdown} 3. How can I participate in the testing phase?
+Let us know by sending an email to [rchelp@northeastern.edu](mailto:rchelp%40northeastern.edu) with the subject “lowpriority partition”. Members of your group will get first access to all newly opened resources for the duration of the testing phase. Only hardware owned by PIs who opt-in will be part of the testing phase.
+::::
+
+::::{dropdown} 4. When will the `lowpriority` partition be officially released?
+Once testing, configuration, and documentation are complete, we will schedule an additional rollout of the `lowpriority` partition and provide access to all users of the HPC cluster. We anticipate this to be in **late 2022/early 2023**. 
+::::
+
+:::{button-link} faq.html
+:color: primary
+:align: center
+Back to the top
+:::
+
+## Miscellaneous
```

Comment:
I think we could just remove this whole section as well. We reference the LPP and Checkpointing pages throughout this entire FAQ.